operator/controllers/kyma_controller.go

```diff
@@ -79,7 +77,7 @@ type KymaReconciler struct {
 
 func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
-	logger.Info("reconciling modules", "resource", req.NamespacedName.String())
```

Comment:
The controller runtime logs already contains following information, don't need to duplicate mention resource.
```
"context":{"controller":"kyma","controllerGroup":"operator.kyma-project.io","controllerKind":"Kyma","Kyma":{"name":"kyma-xin-1-0","namespace":"default"},"namespace":"default","name":"kyma-xin-1-0","reconcileID":"5322de8a-ebfa-4c3a-b5c0-2ac751f80165"}
```